.rubocop.yml

```diff
@@ -139,6 +139,10 @@ Style/SymbolProc:
   Exclude:
     - 'app/models/concerns/hyku_addons/account_behavior.rb'
 
+Style/AndOr:
```

Comment:
You dont need this anymore. 

It's also a bad idea to exclude an entire file from being checked. You should use the comments `# rubocop:disable Rulename` and ` # rubocop:enable Rulename` around the block you are working with.